File: src/Grassland.cpp
#include "Grassland.hpp"

char Grassland::render(){
  if (hasGrass) return '&';
  else return '-';
}


char Grassland::getGrassChar(){return '&';}
/*
Grassland::Grassland(bool hf, bool hg):Cell(0,hf,hg){
} */

File: src Java/Grassland.java
public class Grassland extends Land implements Renderable {
    /**
     * Mengembalikan karakter representasi Grassland
     * @return '&' jika berumput, '-' jika tak berumput
     */
    public char render(){
        if(getHasGrass()){
            return '&';
        } else {
            return '-';
        }
    }

    /**
     * Mengembalikan karakter representasi rumput pada Grassland
     * @return '&'
     */
    public char getGrassChar(){
        return '&';
    }
}
